java-checks/src/test/files/checks/LambdaTooBigCheckCustom.java

```diff
@@ -0,0 +1,41 @@
+class A {
+
+  private void f() {
+    Callable<Integer> c0 = () -> {
```

Comment:
Same comments as earlier. Does not compile. `Callable`, `Predicate`, `List` types are unknown and need to be imported.